packages/expo-modules-core/ios/Tests/EXAppDefinesTest.m

```diff
@@ -0,0 +1,88 @@
+// Copyright 2016-present 650 Industries. All rights reserved.
+
+#import <XCTest/XCTest.h>
+#import <ExpoModulesCore/EXAppDefines.h>
+
+@interface EXAppDefines (EXAppDefinesWithUnloader)
+
++ (void)_unload;
+
+@end
+
+@interface EXAppDefinesTest : XCTestCase
```

Comment:
Can't we use Swift for these tests? 😿

Reply:
i'm not big fans to do that. if tests act like an api document, my preference is to align in the same language. however, we still has a chance to do EXAppDefines swift binding, i'll talk to you today later.